### PR TITLE
Make play whites similar to hand

### PR DESCRIPTION
**Pull request by serainaschraff**

### CODE REVIEW

File: src/components/views/PlayWhites.js [hunk 1]
```diff
@@ -68,75 +71,111 @@ const PlayWhites = () => {
...
 
+  let drawText = "Somehow there don't seem to be any cards today"
+  if(whiteCards) { //If there are cards then display how many are left to draw today
+    if(whiteCards.length>cardsPerHand){
+      drawText= "You can play " +(whiteCards.length) + " more card" + ((whiteCards.length === 1) ? "" : "s") + " today"
+    }
+    else(drawText="No more Cards left to draw today")
+  }
+  if(!blackCardPlayed) { // If the black card has not been played no white cards are available
+    drawText="White Cards will be available once you played a black card"
```
Comment (daveking7) [inline src/components/views/PlayWhites.js:103]: you don't play black cards, only choose/select them?

File: src/components/views/PlayWhites.js [hunk 2]
```diff
@@ -68,75 +71,111 @@ const PlayWhites = () => {
...
-            {cardsContent}
-        </CardContainer>
-        {drawPile}
+      <ViewTitle>
+        Use a White Card to fill in the Blank
+      </ViewTitle>
+      <div className="hand main-container">
+        <div className={"hand card-container"}>
+          <div className="hand text">
+            <h2>Given Black Cards:</h2>
```
Comment (daveking7) [inline src/components/views/PlayWhites.js:168]: typo

File: src/components/views/PlayWhites.js [hunk 3]
```diff
@@ -68,75 +71,111 @@ const PlayWhites = () => {
...
+          </div>
+          {blackCardContent}
+        </div>
+        <div className={"hand card-container"}>
+          <div className="hand text">
+            <h2>Pick a white cards:</h2>
```
Comment (daveking7) [inline src/components/views/PlayWhites.js:174]: typo

File: src/components/views/PlayWhites.js [hunk 4]
```diff
@@ -68,75 +71,111 @@ const PlayWhites = () => {
...
 
     // If no game is left to play on (return value of blackCard is null), this is shown:
-    let blackCardContent =
-        <CardButton className={"card blackCard"} disabled={true}>
-            No black card available
-        </CardButton>
+    if (blackCardPlayed && !blackCard) {
+      blackCardContent = <CardButton className={"card blackCard"}
+                                     onClick={() => {history.push(`/users/${userId}/edit/preferences`)}}>
+        No black card available would you like to change your preferences?
```
Comment (daveking7) [inline src/components/views/PlayWhites.js:131]: add a comma

Comment (daveking7) [reply src/components/views/PlayWhites.js:131]: or even a newline?

File: src/components/views/PlayWhites.js [hunk 5]
```diff
@@ -68,75 +71,111 @@ const PlayWhites = () => {
                 });
         }
 
+        // check if user has played back card if not, black card needs to be played first
+        async function ownBlackCardPlayed() {
+          await api.get(`/users/${userId}/games/blackCards/current`)
+            .then(() => {setBlackCardPlayed(true);})
+            .catch(error => {
+              if (error.response.status === 401 || error.response.status === 404) { setBlackCardPlayed(false); }
```
Comment (daveking7) [inline src/components/views/PlayWhites.js:79]: why do you catch 401 here?

**Review by daveking7 — CHANGES_REQUESTED** [review]
Some typos, but looks nice

File: src/components/views/UserPage.js [hunk 6]
```diff
@@ -147,18 +123,19 @@ const UserPage = () =>{
...
-                </div>
-            </div>
+                    <tr>
+                        <td colSpan="2">
+                            <Button
+                              className="invert"
+                              width="100%"
+                              onClick={() => goToEditPreferences()}
+                            >
+                                Edit Profile
```
Comment (daveking7) [inline src/components/views/UserPage.js:133]: Edit Profile or rather Edit Preferences?

File: src/components/views/PlayWhites.js (hunk 5 above)
Comment (serainaschraff) [reply src/components/views/PlayWhites.js:79]: true makes no sense

**Review by daveking7 — APPROVED** [review]

**Pull request merged**